README.md

```diff
@@ -9,11 +9,14 @@
 * [Glossary](https://wiki.integreat-app.de/glossary)
 
 ## Contributing
```

Comment:
There is also `.github/CONTRIBUTING.md`. Maybe add this information there and link to it here.